etc/schema/mysql/mysql.schema.sql

```diff
@@ -922,8 +940,9 @@ CREATE TABLE history (
   downtime_history_id binary(20) NULL DEFAULT NULL COMMENT 'downtime_history.downtime_id',
   comment_history_id binary(20) NULL DEFAULT NULL COMMENT 'comment_history.comment_id',
   flapping_history_id binary(16) NULL DEFAULT NULL COMMENT 'flapping_history.id',
+  acknowledgement_history_id binary(16) NULL DEFAULT NULL COMMENT 'acknowledgement_history.id',
 
-  event_type enum('notification','state_change','downtime_schedule','downtime_start', 'downtime_end','comment_add','comment_remove','flapping_start','flapping_end') NOT NULL,
+  event_type enum('notification','state_change','downtime_schedule','downtime_start', 'downtime_end','comment_add','comment_remove','flapping_start','flapping_end','acknowledgement_set','acknowledgement_remove') NOT NULL,
```

Comment:
Please rename `acknowledgement_set` and  `acknowledgement_remove` to `ack_set` and `ack_clear`